keycloak/docker-entrypoint.sh

```diff
@@ -5,21 +5,33 @@ if [ -z ${REDIRECT_URIS+x} ]; then
   exit 1
 else
   sed -i "s@{{REDIRECT_URIS}}@$REDIRECT_URIS@g" import/hmda-realm.json
-  echo "Keycloak redirect uris set to $REDIRECT_URIS"
+  echo "Keycloak redirect URIs set to $REDIRECT_URIS"
 fi
 
-if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
+if [ ! -z ${KEYCLOAK_USER+x} ] && [ ! -z ${KEYCLOAK_PASSWORD+x} ]; then
```

Comment:
Blame @wpears. :P  He added that originally on https://github.com/cfpb/hmda-platform-auth/pull/26/files.  I had to look it up.  It is basically a more complete check that the variable is actually set.  I went ahead and switched the others to this format...though I agree that it s a bit of a head-scratcher to look at.

* http://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash/13864829#13864829